src/app/shared/components/main-nav/main-nav.component.scss

```diff
@@ -0,0 +1,62 @@
+@import 'src/app/shared/styles/variables';
+
+:host {
+  ::ng-deep {
```

Comment:
can we avoid ng deep?

Reply:
I had to use ng-deep to customize side-nav material styles

Reply:
ng-deep removed, I used own wrapper instead